core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/rules/SnowflakeCallMapper.scala

```diff
@@ -428,4 +429,51 @@ class SnowflakeCallMapper extends ir.CallMapper with ir.IRHelpers {
     }
   }
 
+  private def arraySort(args: Seq[ir.Expression]): ir.Expression = {
+    makeArraySort(args.head, args.lift(1), args.lift(2))
+  }
+
+  private def makeArraySort(
+      arr: ir.Expression,
+      sortAscending: Option[ir.Expression],
+      nullsFirst: Option[ir.Expression]): ir.Expression = {
+
+    val paramSortAsc = sortAscending.getOrElse(ir.Literal.True)
+    val paramNullsFirst = nullsFirst.getOrElse {
+      paramSortAsc match {
+        case ir.Literal.True => ir.Literal.False
+        case ir.Literal.False => ir.Literal.True
+        case _ => ir.If(ir.Equals(paramSortAsc, ir.Literal.True), ir.Literal.False, ir.Literal.True)
```

Comment:
why do we need this? Per deinition it can only be a literal.

https://docs.snowflake.com/en/sql-reference/functions/array_sort#syntax